BluetoothLEExplorer/BluetoothLEExplorer/Models/GattSampleContext.cs

```diff
@@ -540,7 +540,14 @@ private async void AdvertisementWatcher_Received(BluetoothLEAdvertisementWatcher
                             Windows.UI.Core.CoreDispatcherPriority.Normal,
                             () =>
                             {
-                                d.ServiceCount = args.Advertisement.ServiceUuids.Count();
+                                if (args.Advertisement.ServiceUuids != null)

```

Comment:
if (args.Advertisement.ServcieUuids) is sufficient?

Reply:
I'll follow up in style copy.